meta-mender-imx6ul/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -0,0 +1,3 @@
+# Ensuring system using networkd
+# So that we can add the ethernet configurations
+PACKAGECONFIG_append = " networkd resolved"
```

Comment:
This is not Mender related and should not be here. Actually there are similar definitions for this in [meta-mender-demo](https://github.com/mendersoftware/meta-mender/tree/master/meta-mender-demo), so this would just be redundant.